concepts/comparisons/about.md

```diff
@@ -1,199 +1,291 @@
 # About
 
-A [comparison operator][comparisons] in Python (_also called a Python relational operator_), looks at the values of two operands and returns `True` or `False` based on whether the `comparison` condition is met.
-The most common comparison operators are `"<"`, `">"`, `"=="`, `">="`, `"<="`, and `"!="`.
-They all have the same priority (which is higher than that of the Boolean operations).
+A [comparison operator][comparisons] in Python (_also called a Python relational operator_), looks at the _values_ of two [operands][operand] and returns a boolean `True` or `False` if the `comparison` condition is or is not met.
 
-```python
->>> 7 > 5
-True
->>> 99 < 100
-True
->>> 4 < 4
-False
->>> 4 <= 4
-True
->>> 1 >= 1
-True
->>> 5 == 5
-True
->>> 6 != 6  # not equal to
-False
->>> 5 != 6
-True
->>> not 3 == 3  # interpreted as not(3 == 3)
-False
-```
+The table below shows the most common Python comparison operators:
 
-## Comparison Chaining
 
-Comparisons can be chained arbitrarily, e.g., `x < y <= z` is equivalent to `x < y` `and` `y <= z`, except that `y` is evaluated only once (but in both cases `z` is _not_ evaluated at all when `x < y` is found to be `False`).
-This is also called `short-circuit` evaluation which means the execution is stopped if the truth value of the expression has already been determined.
-Note that the evaluation of expression takes place from left to right.
-In python, short circuiting is supported by various boolean operators, functions and, in this case, comparison chaining.
+| Operator | Operation                | Description                                                    |
+| -------- | ------------------------ | -------------------------------------------------------------- |
+| `>`      | "greater than"             | `a > b` is `True` if `a` is **strictly** greater in value than `b` |
+| `<`      | "less than"                | `a < b` is `True` if `a` is **strictly** less in value than `b`    |
+| `==`     | "equal to"                 | `a == b` is `True` if `a` is **strictly** equal to `b` in value    |
+| `>=`     | "greater than or equal to" | `a >= b` is `True` if `a > b` OR `a == b` in value             |
+| `<=`     | "less than or equal to"    | `a <= b` is `True` if `a < b` or `a == b` in value             |
+| `!=`     | "not equal to"             | `a != b` is `True` if `a == b` is `False`                      |
+| `is`     | "identity"                 | `a is b` is `True` if **_and only if_** `a` and `b` are the same _object_  |
+| `is not` | "negated identity"                 | `a is not b` is `True` if `a` and `b` are **not** the same _object_ |

```

Comment:
If we are alligning for the table we should do so.

Reply:
There's a big difference in the way this appears in code vs render.  When rendered, this is aligned: 

| Operator | Operation                | Description                                                    |
| -------- | ------------------------ | -------------------------------------------------------------- |
| `>`      | "greater than"             | `a > b` is `True` if `a` is **strictly** greater in value than `b` |
| `<`      | "less than"                | `a < b` is `True` if `a` is **strictly** less in value than `b`    |
| `==`     | "equal to"                 | `a == b` is `True` if `a` is **strictly** equal to `b` in value    |
| `>=`     | "greater than or equal to" | `a >= b` is `True` if `a > b` OR `a == b` in value             |
| `<=`     | "less than or equal to"    | `a <= b` is `True` if `a < b` or `a == b` in value             |
| `!=`     | "not equal to"             | `a != b` is `True` if `a == b` is `False`                      |
| `is`     | "identity"                 | `a is b` is `True` if **_and only if_** `a` and `b` are the same _object_  |
| `is not` | "negated identity"                 | `a is not b` is `True` if `a` and `b` are **not** the same _object_ |
| `in`     | "containment test"         | `a in b` is `True` if `a` is member, subset, or element of `b`               |
| `not in` | "negated containment test" | `a not in b` is `True` if `a` is not a member, subset, or element of `b`     |

Reply:
Neverimind.  Prettier to the rescue!

Reply:
Yep, I knew that it would look good rendered (markdown is awesome, tables are cool that way), but having it look good in source is nice too.

The other choice is "less white space" and not align any of it, but have it naturally end, which I do not _not_ like choice as well.

But having it inconsistent, well, that is distracting. ;)